File: web/src/graphql/queries/product.ts
import { gql } from "@apollo/client";

export const PRODUCTS_OF_SHOP = gql`
  query {
    productsOfShop {
      name
      id
      shopId
      amount
      amountType
      price
      updatedAt
      tags {
        id
        name
      }
    }
  }
`;

export const PRODUCT = gql`
  query Product($id: Float!) {
    product(id: $id) {
      name
      id
      shopId
      amount
      amountType
      price
      updatedAt
      tags {
        id
        name
      }
    }
  }
`;

export default PRODUCTS_OF_SHOP;

File: playground_app/lib/graphql/queries/product.dart
String productsOfShop = """
   query {
    productsOfShop {
      name
      id
      shopId
      amount
      amountType
      price
      updatedAt
      tags {
        id
        name
      }
    }
  }
""";

String productQuery = """
query product(\$id: Float!) {
    product(id: \$id) {
      id
      name
      price
      amount
      amountType
      tags {
        id
        name
        description
      }
    }
  }
""";
